app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="scroll_to_current">Scroll to current song</string>
+</resources>
```

Comment:
This is not needed anymore, please revert

Reply:
Ah, kinda missed those two. Removed and squashed my commits into one